sdk/cognitiveservices/Vision.Face/src/Microsoft.Azure.CognitiveServices.Vision.Face.csproj

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <AssemblyTitle>Microsoft Cognitive Services Face SDK</AssemblyTitle>
     <Description>This client library provides access to the Microsoft Cognitive Services Face APIs.</Description>
-    <Version>2.6.0-preview.2</Version>
+    <Version>2.7.0-preview.1</Version>
```

Comment:
Why not 2.6.0-preview.3?

Reply:
This is a good question. This PR includes new detection, recognition, and mask attributes models, which is considered as a minor change. So I think we’d better use a new version number. Please let us know your thoughts. Thank you.